src/librustdoc/html/static/rustdoc.css

```diff
@@ -3,14 +3,18 @@
 	font-family: 'Fira Sans';
 	font-style: normal;
 	font-weight: 400;
-	src: local('Fira Sans'), url("FiraSans-Regular.woff") format('woff');
+	src: local('Fira Sans'),
+		url("FiraSans-Regular.woff2") format("woff2"),
+		url("FiraSans-Regular.woff") format('woff');
```

Comment:
Can you `git rebase` this so as to not include these changes as a merge commit, @jsha? rust-lang/rust uses a no-merge-commit policy (except via bors). https://rustc-dev-guide.rust-lang.org/contributing.html#opening-a-pr

I can walk you through the steps on https://rust-lang.zulipchat.com if you need help.